core/src/main/scala/org/apache/spark/resource/ResourceUtils.scala

```diff
@@ -362,8 +362,13 @@ private[spark] object ResourceUtils extends Logging {
 
   def logResourceInfo(componentName: String, resources: Map[String, ResourceInformation])
     : Unit = {
+    val resourceInfo = if (resources.isEmpty) {
+      s"No extra accelerate resources configured for $componentName."
+    } else {
+      s"Accelerate Resources for $componentName:\n${resources.mkString("\n")}"
```

Comment:
nit - remove Accelerate from here and line 366.  If you want something more specific I would put Custom resources and No custom resources configured just to stay consistent with documentation.

Reply:
addressed, thanks.